docs/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+AB#1234 <!--- Only if relevant, start with a link to an issue on Azure DevOps -->
+
+## Describe your changes
+
+Brief description of your changes - not in-depth because the bulk of the description should be in the task on DevOps.
+
+## Checklist before requesting a review
+
+- [x] I have performed a self-review of my code
+- [x] I have added tests wherever relevant
+- [x] I have made sure that all automated checks pass before requesting a review
+- [x] I have communicated above whether I need any deviation from our PR guidelines (eg. "I don't want the reviewer to merge on my behalf because...")
```

Comment:
```suggestion
- [ ] I have performed a self-review of my code
- [ ] I have added tests wherever relevant
- [ ] I have made sure that all automated checks pass before requesting a review
- [ ] I have communicated above whether I need any deviation from our PR guidelines (eg. "I don't want the reviewer to merge on my behalf because...")
```